tests/test_06.R

```diff
@@ -0,0 +1,26 @@
+# copyright (c) - HU-Berlin / UTHSC / JICS by Danny Arends
+
+# Compare MPCC versus cor() function, mA = 15 x 5, mB = 15 x 10, no missing data
+library(MPCC)
+MPCCinfo()
+
+set.seed(1)
+mAB <- genAB(p = 200, n = 2000, m = 100)
```

Comment:
Generally we keep the test-cases small, since CRAN has a limit of 5 seconds per test
Could you drop n to ~250 ?

Reply:
Ok. Then we'll need to drop n value in test_02.R too.